quickstarts/microsoft.documentdb/cosmosdb-sql-analytical-store/main.bicep

```diff
@@ -0,0 +1,98 @@
+@description('Cosmos DB account name')
+param accountName string = toLower(uniqueString(resourceGroup().id))
+
+@description('Location for the Cosmos DB account.')
+param location string = resourceGroup().location
+
+@description('The name for the database')
+param databaseName string = 'database1'
+
+@description('The name for the container')
+param containerName string = 'container1'
+
+@description('The partition key for the container')
+param partitionKeyPath string = '/partitionKey'
+
+@description('The throughput policy for the container')
+@allowed([
+  'Manual'
+  'Autoscale'
+])
+param throughputPolicy string = 'Autoscale'
+
+@description('Throughput value when using Manual Throughput Policy for the container')
+@minValue(400)
+@maxValue(1000000)
+param manualProvisionedThroughput int = 400
+
+@description('Maximum throughput when using Autoscale Throughput Policy for the container')
+@minValue(4000)
+@maxValue(1000000)
+param autoscaleMaxThroughput int = 4000
+
+@description('Time to Live for data in analytical store. (-1 no expiry)')
+@minValue(-1)
+@maxValue(2147483647)
+param analyticalStoreTTL int = -1
+
+var locations = [
+  {
+    locationName: location
+    failoverPriority: 0
+    isZoneRedundant: false
+  }
+]
+var throughput_Policy = {
+  Manual: {
+    Throughput: manualProvisionedThroughput
+  }
+  Autoscale: {
+    autoscaleSettings: {
+      maxThroughput: autoscaleMaxThroughput
+    }
+  }
+}
+
+resource accountName_resource 'Microsoft.DocumentDB/databaseAccounts@2021-04-15' = {
```

Comment:
remove the _resource from the symbolic names...  I think in this sample you can just use the resource type, eg.

databaseAccount
sqlDatabase
container

Reply:
fixed in new commit.